subtests/docker_cli/build/build.py

```diff
@@ -208,22 +209,26 @@ def parse_postprocess_commands(self, build_def):
         self.logdebug("Postprocessing commands: %s", commands)
         return result
 
-    @staticmethod
-    def basic_postprocess(build_def, command, parameter):
+    def basic_postprocess(self, build_def, command, parameter):
         del parameter  # not used
         if command == 'positive':
-            # Verify zero exit status + healthy output
-            if OutputGood(build_def.dockercmd.cmdresult,
-                          ignore_error=True):
+            # Verify zero exit status and healthy output
+            opg = OutputGood(build_def.dockercmd.cmdresult,
+                             ignore_error=True)
+            if opg:
                 return build_def.dockercmd.cmdresult.exit_status == 0
             else:
+                self.loginfo(str(opg))
```

Comment:
Should this be `warn` (vs info)?

Also, given that there's an identical log below, would you consider adding a differentiator to each one? E.g.

    self.logwarning("OutputGood failed: %s" % opg)
....
    self.logwarning("OutputNotBad failed: %s" % opb)

That makes it easier for a human to determine which of those two logs triggered, and continue debugging from there.